### ISSUE TITLE
prevButton and nextButton should have text or aria-label

### ISSUE THREAD

**Issue by helgefredheim** [gpbl/react-day-picker#258]
Great component! 

An issue after having tested accessibility with a blind person: The previous and next buttons should have `aria-label="Previous | Next"` (localizeable) or a text: 

`<span role="button" {...}>Previous</span>`

This is due to screen reader accessibility. Without the `aria-label` or text, the button is read as something weird.

**Comment by gpbl** [comment]
Thanks for the report! This will be added soon. Meanwhile, you can create your own navbar:

http://react-day-picker.js.org/examples/?customElements

**Comment by helgefredheim** [comment]
So I did, thank you!

**Comment by gpbl** [comment]
Published as v5.1.1. Check the new [labels](http://react-day-picker.js.org/APIProps.html#labels) prop!
